### PR TITLE
Finalize Rust frontend test

### PR DESCRIPTION
**Pull request by robinmaisch**
Concerning #634:
The test is now finalized. The acceptable baseline coverage had to be reduced from 90% (which other frontends achieve) to 75% due to the nature of the language and the content and formatting of the test files.

The test should now only fail if the token extraction process is interfered with, e.g. during maintenance or evolution.

### CODE REVIEW

**Review by tsaglam — APPROVED** [review]

File: jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java [hunk 1]
```diff
@@ -86,15 +87,15 @@ private void testSourceCoverage(String fileName, TokenList tokens) {
             } else {
                 logger.info("Coverage: %.1f%%.".formatted(coverage * 100));
                 logger.info("Missing lines {}", codeLines);
-                if (coverage - 0.9 <= EPSILON) {
-                    // TODO use fail() instead when frontend is ready
+                if (coverage - BASELINE_COVERAGE <= EPSILON) {
                     logger.error("Source coverage is unsatisfactory");
+                    assertTrue(false);
```
Comment (JanWittler) [inline jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java:92]: Closing this was maybe a bit too fast...
This should be `assertEquals("source coverage is only <format coverage> but should be <format expected coverage>", BASELINE_COVERAGE, coverage, EPSILON)`

Comment (robinmaisch) [reply jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java:92]: The method that you refer to checks that the absolute difference is smaller than a delta, right? `coverage` is supposed to be greater than or equal to `BASELINE_COVERAGE`.

Comment (JanWittler) [reply jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java:92]: Good point. But the if-condition should still be inlined to the assert statement.
`assertTrue(<description>, coverage - BASELINE_COVERAGE > EPSILON)`

Comment (robinmaisch) [reply jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java:92]: Ah, much better, that's right. I pushed this version now. I need to open another PR, right?

Comment (JanWittler) [reply jplag.frontend.rust/src/test/java/de/jplag/rust/RustFrontendTest.java:92]: yes

**Pull request merged**